io.github.CoyoteTM.json

```diff
@@ -0,0 +1,93 @@
+{
+    "app-id" : "io.github.CoyoteTM",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "19.08",
+    "sdk" : "org.freedesktop.Sdk",
+    "command" : "coyotetm",
+    "tags" : [
+        "nightly"
+    ],
+    "desktop-file-name-prefix" : "(Nightly) ",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--talk-name=org.gtk.vfs",
```

Comment:
this is not needed (should be removed). It was never used.

Reply:
Also are you really using gvfs since it's a fltk app?

Reply:
fixed